parley/src/style/mod.rs

```diff
@@ -41,6 +43,42 @@ pub enum OverflowWrap {
     BreakWord,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum LineHeight {
+    /// Line height specified relative to what's defined in the font's metrics.
```

Comment:
```suggestion
    /// The line's height is a multiple of the ["line height"](https://developer.apple.com/library/archive/documentation/StringsTextFonts/Conceptual/TextAndWebiPhoneOS/TypoFeatures/TextSystemFeatures.html) defined by the font's metrics.
```
There might be a better link for Line height available. The picture in https://tex.stackexchange.com/questions/614909/pk-height-ascend-descend-and-cap-height-of-a-pk-font-face is pretty good.

Reply:
Not sure how I feel about linking to a mostly-unrelated StackOverflow post about Go, and couldn't find any other diagrams that mentioned the line gap / leading metric. I just explained it in the doc comment.